coeus-python/Cargo.toml

```diff
@@ -11,6 +11,7 @@ crate-type = ["cdylib"]
 [dependencies]
 pyo3 = { version = "0.16.5", features = ["extension-module", "abi3-py37"] }
 coeus = {version = "0.1.1", path = "../coeus"}
+abxml = {version = "0.8.2", default-features = false}
```

Comment:
This is not needed as far as I can see?

Reply:
(see that there is a `decode_function` https://github.com/UbiqueInnovation/coeus-rs/blob/main/coeus/coeus_models/src/models/files.rs#L53)